edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceScopeIdentitiesCache.cs

```diff
@@ -340,59 +348,79 @@ async Task RefreshCache()
         {
             while (true)
             {
-                var currentCacheIds = new List<string>();
-
-                try
+                bool succeeded = await this.RefreshCacheInternal();
+                if (this.ShouldRetryRefreshCache(succeeded))
+                {
+                    Events.RetryRefreshCycle(this.refreshDelay);
+                    await this.IsReady(this.refreshDelay);
```

Comment:
>this.refreshDelay [](http://example.com/codeflow?start=39&length=17)

This delay value might not be appropriate for the retry loop. It's user configurable via env var, and the customer facing name might not give clear indication that it'll affect something as obscure as this.

maybe it's better if we just hardcode this to 2min, or maybe even do the traditional fallback retry.

Reply:
I added a separate delay and set to 1min by default